src/commands/toggleValueVisability.ts

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AppSettingTreeItem } from "vscode-azureappservice";
+
+export async function toggleValueVisability(node: AppSettingTreeItem): Promise<void> {
+    node.toggleValueVisability();
+    await node.refresh();
```

Comment:
Shouldn't `toggleValueVisability` handle the refresh?

Reply:
Also if this ends up being one line of code, I think I'm fine with it going directly in extension.ts

Reply:
Good point